.storybook/webpack.config.js

```diff
@@ -11,8 +11,8 @@ const webpack = require('webpack');
 module.exports = {
     plugins: [
         new webpack.DefinePlugin({
-            API_HOST: JSON.stringify(`${process.env.CLIENT_API_URL}:${process.env.CLIENT_PORT}`),
-            LOGIN_URL: JSON.stringify(`${process.env.CONTINUUM_LOGIN_URL}:${process.env.CONTINUUM_LOGIN_PORT}/submit`),
+            API_HOST: JSON.stringify(`${process.env.CLIENT_API_URL}`),
```

Comment:
Why the `JSON.stringify()` ?

Reply:
otherwise we would havesay:
`API_HOST: http://localhost:3000`

instead of 
`API_HOST: "http://localhost:3000"`

Reply:
Ah you mean its for the string quotation? E.g.
```
> JSON.stringify(process.env.EDITOR)
'"vim"'
> process.env.EDITOR
'vim'
```

Reply:
yup